esrally/driver/driver.py

```diff
@@ -468,15 +468,15 @@ def __init__(self):
 
     def receiveMsg_PoisonMessage(self, poisonmsg, sender):
         self.logger.error("Track Preparator received a fatal indication from a load generator (%s). Shutting down.", poisonmsg.details)
-        self.send(self.original_sender, actor.BenchmarkFailure("Fatal track preparation indication", poisonmsg.details))
+        self.send(self.driver_actor, actor.BenchmarkFailure("Fatal track preparation indication", poisonmsg.details))
 
     @actor.no_retry("track preparator")  # pylint: disable=no-value-for-parameter
-    def receiveMsg_Bootstrap(self, msg, sender):
+    def receiveMsg_Bootstrap(self, msg, driver_actor):
         # load node-specific config to have correct paths available
         self.cfg = load_local_config(msg.config)
         # this instance of load_track occurs once per host, so install dependencies if necessary
         load_track(self.cfg, install_dependencies=False)
-        self.send(sender, ReadyForWork())
+        self.send(driver_actor, ReadyForWork())
```

Comment:
What are your thoughts about setting `self.driver_actor = sender` here instead of `receiveMsg_PrepareTrack` since it's received first?

Reply:
That would make more sense, I hadn't thought about that